site2/website/scripts/replace.js

```diff
@@ -178,13 +178,20 @@ for (v of versions) {
       `${v}`,
       binaryReleaseUrl(`${v}`),
       connectorReleaseUrl(`${v}`),
-      prestoPulsarReleaseUrl(`${latestVersion}`),
+      offloaderReleaseUrl(`${v}`),
+      prestoPulsarReleaseUrl(`${v}`),
       downloadPageUrl(),
       rpmReleaseUrl(`${v}`, ""),
       rpmReleaseUrl(`${v}`, "-debuginfo"),
       rpmReleaseUrl(`${v}`, "-devel"),
       debReleaseUrl(`${v}`, ""),
       debReleaseUrl(`${v}`, "-dev"),
+
+      rpmDistUrl(`${v}`, ""),
+      rpmDistUrl(`${v}`, "-debuginfo"),
+      rpmDistUrl(`${v}`, "-devel"),
+      debDistUrl(`${v}`, ""),
+      debDistUrl(`${v}`, "-dev")
```

Comment:
You can try:

```
      `${vWithoutIncubating}`,
      `${v}`,
      binaryReleaseUrl(`${v}`),
      connectorReleaseUrl(`${v}`),
      offloaderReleaseUrl(`${v}`),
      prestoPulsarReleaseUrl(`${v}`),
      downloadPageUrl(),
      rpmDistUrl(`${v}`, ""),
      rpmDistUrl(`${v}`, "-debuginfo"),
      rpmDistUrl(`${v}`, "-devel"),
      debDistUrl(`${v}`, ""),
      debDistUrl(`${v}`, "-dev"),
      rpmDistUrl(`${v}`, ""),
      rpmDistUrl(`${v}`, "-debuginfo"),
      rpmDistUrl(`${v}`, "-devel"),
      debDistUrl(`${v}`, ""),
      debDistUrl(`${v}`, "-dev")
```
You can test the results by opening the html file `build/pulsar/docs/en/2.4.1/client-libraries-cpp/index.html` through the browser. I think this is correct.

Reply:
I'v fixed it, PTAL.